go.mod

```diff
@@ -15,3 +17,6 @@ require (
 	gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c
 	gopkg.in/yaml.v3 v3.0.1
 )
+
+// Temporary workaround until https://github.com/google/gnostic-models/pull/5 is merged
+replace github.com/google/gnostic-models => github.com/pkwarren/gnostic-models v0.0.0-20230713215956-94610bf2de2f
```

Comment:
This PR is a draft until https://github.com/google/gnostic-models/pull/5 can land (and update to a new version of gnostic-models).

Reply:
Updated to mark this as ready for review in case that indicates it is ready to go. Before merging this will require the above PR to be merged/released first.